datasets/uy/pep/crawler.py

```diff
@@ -26,19 +27,34 @@ def parse_sheet_row(context: Context, row: Dict[str, str]):
     id_number = row.pop("c-i")
     person_name = row.pop("nombre")
     role = row.pop("cargo")
+    organization_name = row.pop("organismo")
 
     person.id = context.make_slug(id_number, prefix="uy-cedula")
     person.add("idNumber", id_number)
     person.add("name", person_name)
 
-    position = h.make_position(context, role, country="uy", lang="spa")
+    position = h.make_position(
+        context, f"{role}, {organization_name}", country="uy", lang="spa"
+    )
+
+    categorisation = categorise(context, position)
+    if not categorisation.is_pep:
+        return
+
+    occupancy = h.make_occupancy(
+        context,
+        person,
+        position,
+    )
 
     context.emit(person, target=True)
     context.emit(position)
+    context.emit(occupancy)
```

Comment:
could you do context.audit_data() on row after this? then we'll get a little warning if they add more columns, prompting us to consider including that in the crawler in future